translations/de.ts

```diff
@@ -1132,12 +1132,12 @@ instead of system taskbar.</source>
         <location filename="../src/widget/form/settings/generalsettings.ui" line="370"/>
         <source>Always notify about new messages in groupchats.</source>
         <comment>toolTip for Group chat always notify</comment>
-        <translation>Immer auf neue Nachrichten in Gruppenchats hinweisen.</translation>
+        <translation>Du wirst benachrichtigt, wenn neue Nachrichten in Gruppenchats eintreffen.</translation>
     </message>
     <message>
         <location filename="../src/widget/form/settings/generalsettings.ui" line="373"/>
         <source>Group chats always notify</source>
-        <translation>Gruppenchat-Hinweise</translation>
+        <translation>Gruppenchat-Benachrichtigungen</translation>
```

Comment:
Korrekt wäre diese Übersetzuing keineswegs. Sie wäre eher eine Wort-für-Wort-Übersetzung.
"immer einschalten" ist redundant. Immer wenn es eingeschalten ist ist es eingeschalten?

Die Option "Pseudo-Offline-Nachrichten" heißt auch nicht "Pseudo-Offline-Nachrichten einschalten". Es wäre äußerst unintuitive, wenn das Häkchen bei offline messages" diese deaktivieren würde, statt sie zu aktivieren. 
Soll heißen: Es ist durchaus üblich "einschalten", "aktivieren" u.ä. wegzulassen, da dies in  kontextueller Analogue gefolgert wird (Option mit diesem Namen aktivieren, bedeutet diese Funktion aktivieren"
"helles Icon" heißt auch nicht "Helles Icon auswählen/anzeigen/aktivieren/immer aktivieren"
In der Kürze liegt die Würze.